mix.exs

```diff
@@ -54,6 +54,7 @@ defmodule Neuron.Mixfile do
     [
       source_ref: "v#{@version}",
       main: "Neuron",
+logo: "neuron/logo/logo.png",
```

Comment:
Could you move this to the same level of indentation as the rest of the items in this list?